src/main/java/sirius/biz/storage/layer2/L3Uplink.java

```diff
@@ -132,6 +132,14 @@ protected VirtualFile createPlaceholder(@Nullable Directory parentDirectory, Vir
             file.attach(new Placeholder(parent, name));
```

Comment:
file.attach(parentDirectory.getStorageSpace());

and also attach the StorageSpace in wrapDirectory and wrapBlob

Reply:
and then add a method either in L3Uplink or VirtualFileSystem or StorageUtils (for better visibitilty) 
isWorkFile(VirtualFile file) { return WORK.equals(file.tryAs(StorageSpace.class).map(StorageSpace::getName).orElse("")); }

Reply:
Or, instead of checking against the name, we could add a method to BlobStorageSpace "isAutoCleanup" which checks for retentionDays > 0 which would be true for work and tmp but not for others...

Reply:
Closing again, since lots will be reverted making reviews more difficult.